reactive_graph/src/owner/arc_stored_value.rs

```diff
@@ -0,0 +1,126 @@
+use crate::{
+    signal::guards::{Plain, ReadGuard, UntrackedWriteGuard},
+    traits::{DefinedAt, IsDisposed, ReadValue, WriteValue},
+};
+use std::{
+    fmt::{Debug, Formatter},
+    hash::Hash,
+    panic::Location,
+    sync::{Arc, RwLock},
+};
+
+/// A reference-counted getter for any value non-reactively.
+///
+/// This is a reference-counted signal, which is `Clone` but not `Copy`.
```

Comment:
nit: Here and elsewhere (in the new StoredValue traits), these docs refer to these as "signals" but they're really not; I'd probably just replace "signal" with "value".

(Presumably just copy-pasted from the other signal docs)